index.js

```diff
@@ -55,15 +55,66 @@ const sendEmail = async ({
       expiredKeys,
     }),
   }
-  sgMail.send(msg)
+  //sgMail.send(msg)
 }
 
+const getKeyDetails = async ({
+  keyNames
+}) => {
+  const keyInfoFileContents = await readFileAsync(join(__dirname, './keys.info'))
+  var keysToReturn = {}
+  keyListInfo = JSON.parse(keyInfoFileContents)
+  keyListInfo.keys.forEach((keyInfo) => {
+    if (keyNames.indexOf(keyInfo.name) != -1) {
+      keysToReturn[keyInfo.name] = keyInfo
+    }
+  })
+  return keysToReturn
+}
+
+const sendRemindersForExpiredKeys = async({
+  expiredKeys
+}) => {
+  console.log(expiredKeys)
+  expiredKeyMailTemplate = await readFileAsync(join(__dirname, './reminderemail.html'))
+  keyListInfo = await getKeyDetails({
+    keyNames: expiredKeys,
+  })
+  
+  expiredKeys.forEach((expiredKey) => {
+    
+    var compiledExpiredEmail = compile(expiredKeyMailTemplate.toString())
```

Comment:
`const` and `let` are preferable in javascript to `var` with modern js. In these cases, I would use `const` 😄 

More background: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75